plugins/modules/appliance_access_consolecli.py

```diff
@@ -106,8 +106,12 @@
 }  # pylint: disable=line-too-long
 
 from ansible.module_utils.basic import env_fallback
+import os
 
 try:
+    if not os.getenv("VMWARE_ENABLE_TURBO", False):
+        raise ImportError()
+
```

Comment:
I'm not sure about this. Looks a bit like misusing the ImportError exception here because, well, we don't have a problem importing something. Or do we? This doesn't look too great to me.

Although this means some additional lines, I think this would be better:

```
if os.getenv("VMWARE_ENABLE_TURBO", True):
    try:
        ...
    except ImportError:
        from ansible.module_utils.basic import AnsibleModule
else:
    from ansible.module_utils.basic import AnsibleModule
```

This would make the logic clearer.

It would be great to hear the opinion of other people about this! After all, I might be wrong there.